src/semanticUiReactReExport.ts

```diff
@@ -0,0 +1,195 @@
+
+export {
+  default as Sticky,
+  StickyProps,
+  StrictStickyProps,
+} from 'semantic-ui-react/dist/commonjs/modules/Sticky'
+
+export {
+  default as Grid,
+  GridProps,
+  StrictGridProps,
+} from 'semantic-ui-react/dist/commonjs/collections/Grid'
+
+export {
+  default as Accordion,
+  AccordionProps,
+  StrictAccordionProps,
+} from 'semantic-ui-react/dist/commonjs/modules/Accordion/Accordion'
+
+export {
+  default as Segment,
+  SegmentProps,
+  StrictSegmentProps,
+} from 'semantic-ui-react/dist/commonjs/elements/Segment'
+
+export {
+  default as Loader,
+  LoaderProps,
+  StrictLoaderProps,
+} from 'semantic-ui-react/dist/commonjs/elements/Loader'
+
+export {
+  default as Modal,
+  ModalProps,
+  StrictModalProps,
+} from 'semantic-ui-react/dist/commonjs/modules/Modal'
+
+export {
+  default as Message,
+  MessageProps,
+  MessageSizeProp,
+  StrictMessageProps,
+} from 'semantic-ui-react/dist/commonjs/collections/Message'
+
+export {
+  default as TextArea,
+  TextAreaProps,
+  StrictTextAreaProps,
+} from 'semantic-ui-react/dist/commonjs/addons/TextArea'
+
+export {
+  default as FormRadio,
+  FormRadioProps,
+  StrictFormRadioProps,
+} from 'semantic-ui-react/dist/commonjs/collections/Form/FormRadio'
+
+export {
+  default as Card,
+  CardProps,
+  StrictCardProps,
+} from 'semantic-ui-react/dist/commonjs/views/Card'
+
+export {
+  default as Radio,
+  RadioProps,
+  StrictRadioProps,
+} from 'semantic-ui-react/dist/commonjs/addons/Radio'
+
+export {
+  default as FormCheckbox,
+  FormCheckboxProps,
+  StrictFormCheckboxProps,
+} from 'semantic-ui-react/dist/commonjs/collections/Form/FormCheckbox'
+
+export {
+  default as Pagination,
+  PaginationProps,
+  StrictPaginationProps,
+} from 'semantic-ui-react/dist/commonjs/addons/Pagination'
+
+export {
+  default as Search,
+  SearchProps,
+  SearchResultData,
+  StrictSearchProps,
+} from 'semantic-ui-react/dist/commonjs/modules/Search'
+
+export {
+  default as Menu,
+  MenuProps,
+  StrictMenuProps,
+} from 'semantic-ui-react/dist/commonjs/collections/Menu'
+
+export {
+  default as Dimmer,
+  DimmerProps,
+  StrictDimmerProps,
+} from 'semantic-ui-react/dist/commonjs/modules/Dimmer'
+
+export {
+  default as Popup,
+  PopupProps,
+  StrictPopupProps,
+} from 'semantic-ui-react/dist/commonjs/modules/Popup'
+
+export {
+  default as Divider,
+  DividerProps,
+  StrictDividerProps,
+} from 'semantic-ui-react/dist/commonjs/elements/Divider'
+
+export {
+  default as TableCell,
+  TableCellProps,
+  StrictTableCellProps,
+} from 'semantic-ui-react/dist/commonjs/collections/Table/TableCell'
+
+export {
+  default as TableHeaderCell,
+  TableHeaderCellProps,
+  StrictTableHeaderCellProps,
+} from 'semantic-ui-react/dist/commonjs/collections/Table/TableHeaderCell'
+
+export {
+  default as Checkbox,
+  CheckboxProps,
+  StrictCheckboxProps,
+} from 'semantic-ui-react/dist/commonjs/modules/Checkbox'
+
+export { SemanticCOLORS, SemanticICONS } from 'semantic-ui-react/src/generic'
```

Comment:
Trying to understand what is this file for and where it's used. In `ApplicationViewWrapper` it seems to be importing straight from `semantic-ui-react`: `import { SemanticICONS } from 'semantic-ui-react'`